src/core/live-announcer/live-announcer.ts

```diff
@@ -0,0 +1,52 @@
+import {Injectable} from 'angular2/core';
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Injectable()
+export class MdLiveAnnouncer {
+
+  private announcerEl: Element;
+
+  constructor() {
+    this.announcerEl = this._getAnnouncerElement();
+  }
+
+  /**
+   * @param message Message to be announced to the screenreader
+   * @param politeness The politeness of the announcer element.
+   */
+  announce(message: string, politeness = 'polite'): void {
+    this.announcerEl.textContent = '';
+
+    this.announcerEl.setAttribute('aria-live', politeness);
```

Comment:
I think we'll need to validate that a changing politeness level is correctly utilized by Assistive Technologies.